### PR TITLE
Fix Bug #419

### PR DESCRIPTION
**Pull request by adraut**
Fix Bug #419

### CODE REVIEW

**Comment by carloscds** [comment]
@adraut Hi, can you change your PR moving to vnext branch ?

**Comment by adraut** [comment]
@carloscds I sure can.

**Comment by adraut** [comment]
Closing because it was the wrong branch.

**Comment by giggio** [comment]
@carloscds @adraut This is a bug fix, it should go on the `master` branch.

File: src/CSharp/CodeCracker/Usage/RemoveUnusedVariablesCodeFixProvider.cs [hunk 1]
```diff
@@ -24,8 +24,18 @@ public class RemoveUnusedVariablesCodeFixProvider : CodeFixProvider
             var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
             var diagnostic = context.Diagnostics.First();
             var diagnosticSpan = diagnostic.Location.SourceSpan;
-            var variableUnused = root.FindToken(diagnosticSpan.Start).Parent.AncestorsAndSelf().OfType<LocalDeclarationStatementSyntax>().First();
-            context.RegisterCodeFix(CodeAction.Create($"Remove unused variable : '{ variableUnused.Declaration.Variables.First()}'", c => RemoveVariableAsync(context.Document, variableUnused, c), nameof(RemoveUnusedVariablesCodeFixProvider)), diagnostic);
+            var parent = root.FindToken(diagnosticSpan.Start).Parent;
+
+            if (parent.RawKind != (int)Microsoft.CodeAnalysis.CSharp.SyntaxKind.CatchDeclaration)
```
Comment (giggio) [inline src/CSharp/CodeCracker/Usage/RemoveUnusedVariablesCodeFixProvider.cs]: Use `IsKind` instead of `RawKind`.

**Comment by adraut** [comment]
@giggio The change to IsKind has been pushed.

**Pull request merged**